WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -3678,6 +3678,11 @@ public void onMediaUploaded(OnMediaUploaded event) {
             return;
         }
 
+        if (!NetworkUtils.isNetworkAvailable(this)) {
```

Comment:
I worry that if we do not check for an error as well, we might erroneously mark successfully uploaded media as paused. WDYT?